test/integration_test.go

```diff
@@ -41,7 +41,7 @@ func runIntegration() (string, string, error) {
 		"-scrape_interval", "15s",
 		"-exporter_bind_address", "127.0.0.1",
 		"-exporter_bind_port", "9183",
-		"-allow_regex", "^*$",
+		"-filter_list", "regex \"^*$\"",
```

Comment:
@gsanchezgavier in the example there is a `windowsService.name:` that I do not see used here, is it normal?

Reply:
Wasn't needed , but that's going to change when that parameter `windowsService.name` is enforced